packages/wallet-adapter-react/src/components/AboutAptosConnect.tsx

```diff
@@ -16,7 +16,7 @@ import { Web3Graphic } from "../graphics/Web3Graphic";
 import { HeadlessComponentProps, createHeadlessComponent } from "./utils";
 
 export const EXPLORE_ECOSYSTEM_URL =
-  "https://aptosfoundation.org/ecosystem/projects/all";
+  "https://aptosnetwork.com/ecosystem/directory/category/defi";
 
 const AboutAptosConnectContext = createContext<{
```

Comment:
should we also rename? also rename the file 
```suggestion
const AboutPetraWebContext = createContext<{
```

Reply:
I think we need to be consistent with the changes for users as well as for devs. There are user facing changes and dev facing changes, the later introduces some breaking changes. Perhaps we can break it into 2 releases, user facing changes can go into a minor version bump, and dev facing can go into major version bump